<launch>
    <!-- PS3 controller node -->
    <node respawn="true" pkg="joy" type="joy_node" name="PS3_joy" >
        <param name="dev" type="string" value="/dev/input/js0" />
        <param name="deadzone" value="0.25" />
    </node>

    <!-- ps3 teleoperation -->
    <param name="speed" type="double" value="0.65" />
    <param name="angular_speed" type="double" value="1.5" />
    <node pkg="rover" type="rover_teleop_joy" name="teleop" />
</launch>
